src/test/java/com/snowflake/kafka/connector/SnowflakeSinkTaskForStreamingIT.java

```diff
@@ -162,31 +162,39 @@ public void testTaskToClientMapping() throws Exception {
 
   @Test
   public void testMultipleSinkTaskWithLogs() throws Exception {
-    int partition0 = 0;
-    int partition1 = 1;
-
     // setup log mocking for task1
     MockitoAnnotations.initMocks(this);
     Mockito.when(logger.isInfoEnabled()).thenReturn(true);
     Mockito.when(logger.isDebugEnabled()).thenReturn(true);
     Mockito.when(logger.isWarnEnabled()).thenReturn(true);
 
+    // TODO @rcheng: factor these out, has been a pain updating this test with the dupe code...
     // set up configs
     String task0Id = "0";
+    int partition0 = 0;
+    String topicName0 = "topic0";
     Map<String, String> config0 = TestUtils.getConfForStreaming();
+    config0.put(BUFFER_COUNT_RECORDS, "1"); // override
+    config0.put(INGESTION_METHOD_OPT, IngestionMethodConfig.SNOWPIPE_STREAMING.toString());
     SnowflakeSinkConnectorConfig.setDefaultValues(config0);
     config0.put(Utils.TASK_ID, task0Id);
+    TopicPartition topicPartition0 = new TopicPartition(topicName0, partition0);
 
     String task1Id = "1";
-    int taskOpen1Count = 0;
+    int partition1 = 1;
```

Comment:
ok i thiiink this can be 0? this will be updated in the next PR, but happy to try updating it here as well

Reply:
I am okay coupling all fixes here. Unless they are changing something big.